docs/EXTENDED-ARCHIVES.md

```diff
@@ -0,0 +1,428 @@
+Etcher Extended Archives
+========================
+
+Etcher extended archives are archive files including an image and a special
+metadata directory including information that describes the image, device type,
+and vendor in detail, enabling Etcher to make use of such information to
+deliver advanced features and a more pleasant experience to the user.
+
+Entities
+--------
+
+### Vendor
+
+A vendor represents a manufacturer of a device type and potential publisher.
+
+*Some examples of image vendors are: Beagleboard, Raspberry Pi Foundation,
+Intel, and ODroid.*
+
+### Publisher
+
+A publisher represents an entity that maintains and publishes official images.
+
+*Some examples of image publishers are: Tizen, RetroPie, Resin.io, and CoreOS.*
+
+### Device
+
+A device represents a specific device type manufactured by a vendor.
+
+*Some examples of devices are: Beaglebone Black, Beaglebone Green, Intel
+Edison, and Raspberry Pi 3.*
+
+### Client
+
+A client represents the program making use of the information about the other
+entities.
+
+*At the moment of this writing, the only existent client is the Etcher application.*
+
+File format
+-----------
+
+The file format should have the following characteristics:
+
+- Streamability: The extended archive allows the client to consume all its
+  information in a streaming fashion
+
+- Compression: The extended archive should be compressed in order to save
+  network resources
+
+We base this format on the [`tar`][tar] archive format, which has the following
+benefits:
```

Comment:
Oh? This is a major change. I thought the idea was to use `.zip` format, so that extended-archive files where still usable by people on Windows (which has built-in `.zip` support), but who don't (for whatever reason) want to use Etcher?